modules/data-table/main/theme.scss

```diff
@@ -51,9 +51,11 @@
     }
   }
 
-  .hx-data-table-footer {
+  .hx-data-table-control-panel-bottom,
+  .hx-data-table-control-panel-compact,
+  .hx-data-table-control-panel {
```

Comment:
There is no longer a concept of a 'footer' as it can appear above or below the table
There is the compact control panel that houses the 'menu' button and a paginator, the main control panel that contains everything and the bottom control panel that contains a paginator and a page size picker